rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/QueryHelper.java

```diff
@@ -0,0 +1,80 @@
+package pro.taskana.common.rest;
+
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.util.MultiValueMap;
+
+import pro.taskana.common.api.BaseQuery.SortDirection;
+import pro.taskana.common.api.exceptions.InvalidArgumentException;
+import pro.taskana.common.internal.util.CheckedBiConsumer;
+
+public class QueryHelper {
```

Comment:
I don't like this name. 
Any suggestion? 
I'd love to expand this class in the future by adding e.g. the extraction of paging here, etc.

Reply:
What's wrong with the name? Maybe we should discuss this if you have concerns.